Makefile

```diff
@@ -30,31 +37,38 @@ po-push: po/cockpit-composer.pot $(WEBLATE_REPO)
 	git -C $(WEBLATE_REPO) commit -m "Update source file" -- cockpit-composer.pot
 	git -C $(WEBLATE_REPO) push
 
-po/cockpit-composer.pot: npm-install
+po/cockpit-composer.pot: $(WEBPACK_TEST)
 	NODE_ENV=$(NODE_ENV) npm run translations:extract
 	NODE_ENV=$(NODE_ENV) npm run translations:json2pot
 
-npm-install:
-	npm install
-
 install: all
 	mkdir -p /usr/share/cockpit/composer
 	cp -r public/* /usr/share/cockpit/composer
 	mkdir -p /usr/share/metainfo/
 	cp io.weldr.cockpit-composer.metainfo.xml /usr/share/metainfo/
 
 # this requires a built source tree and avoids having to install anything system-wide
-devel-install:
+devel-install: $(WEBPACK_TEST)
 	mkdir -p ~/.local/share/cockpit
 	ln -s `pwd`/public/dist ~/.local/share/cockpit/composer
 
-dist-gzip: NODE_ENV=production
-# package test dependence to avoid clone or fetch
-dist-gzip: all $(PACKAGE_NAME).spec machine test/common
-	mkdir -p $(PACKAGE_NAME)-$(VERSION)
-	cp -r public/ test/ LICENSE.txt README.md $(PACKAGE_NAME).spec io.weldr.cockpit-composer.metainfo.xml $(PACKAGE_NAME)-$(VERSION)
-	tar -czf $(PACKAGE_NAME)-$(VERSION).tar.gz $(PACKAGE_NAME)-$(VERSION)
-	rm -rf $(PACKAGE_NAME)-$(VERSION)
+
+dist-gzip: $(TARFILE)
```

Comment:
Nice, as a side issue this should also fix `make` to not always rebuild everything even if nothing changed.